api/ReadableStreamBYOBReader.json

```
@@ -213,6 +213,55 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options_min_parameter": {
+          "__compat": {
+            "description": "<code>options.min</code> parameter",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "impl_url": "https://crbug.com/40942083"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
```

Based on the resolved bug, it's in Nightly:

```suggestion
                "version_added": "preview",
```